package-lock.json

```diff
@@ -0,0 +1,13392 @@
+{
+  "name": "panel",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
+  "requires": true,
```

Comment:
Let's not add a package-lock.json file, since yarn uses yarn.lock!

Reply:
Oops, I'm not sure how that got file got added/generated.